Matlab/syntax_test_matlab.m

```diff
@@ -201,3 +201,9 @@ function foo(bar)
 %   ^^^^^^^^ string.quoted.double.matlab
 %           ^ punctuation.definition.string.end.matlab
 
+%---------------------------------------------
+% parfor keyword test
+% parfor
+parfor x = 1:10
+^ keyword.control.matlab
```

Comment:
Syntax test lines need to begin with a certain token that is defined in the first line of the file (and is usually the line comment punctuation, I. E. `%` here).

Reply:
Thanks! Fixed now I think.